src/runner.ts

```diff
@@ -59,21 +58,25 @@ export class Runner {
     manipulators?: Manipulator[],
     emitter?: Emitter
   ) {
-    if (!project && !args) {
+    this.parser = parser || new Parser();
```

Comment:
let's add before constructor on if arg is undefined, what is the default value.

Reply:
Created default args